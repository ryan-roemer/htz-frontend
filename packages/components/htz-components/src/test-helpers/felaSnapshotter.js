import React from 'react';
import createRenderer from '../components/StyleProvider/createRenderer';
import StyleProvider from '../components/StyleProvider/StyleProvider';
import createFelaSnapshotter from './createFelaSnapshotter';

const testRenderer = createRenderer({ isRtl: true, });
const felaSnapshotter = createFelaSnapshotter(StyleProvider, testRenderer);

export default felaSnapshotter;
