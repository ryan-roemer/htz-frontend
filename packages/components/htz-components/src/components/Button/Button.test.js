import React from 'react';
import felaSnapshotter from '../../test-helpers/felaSnapshotter';
import StyledButton from './Button';

it('renders correctly', () => {
  const tree = felaSnapshotter(<StyledButton>Click here</StyledButton>);
  expect(tree).toMatchSnapshot();
});
