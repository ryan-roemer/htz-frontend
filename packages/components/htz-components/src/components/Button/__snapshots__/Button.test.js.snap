// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "component": <div>
    <button
        className="htz-a"
    >
        Click here
    </button>
</div>,
  "styles": ".htz-a{outline:1px solid #09a5d9}",
}
`;
